### PR TITLE
Added support for i18n

### PR DESCRIPTION
**Pull request by clintonb**
The project now supports i18n with translations from Transifex.

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @KostyaEsmukov. 

I setup a Transifex project at https://www.transifex.com/django-waffle/django-waffle/.

**Comment by KostyaEsmukov** [comment]
Wow, thanks for the prompt implementation! I've accepted the Transifex invitation and I'm going to add translations soon.

**Review by KostyaEsmukov — COMMENTED** [review]
Could you please add `verbose_name` and `verbose_name_plural` to all models? Also having `verbose_name` on all model fields would be great.

Thanks!

File: waffle/models.py
Comment (KostyaEsmukov) [inline waffle/models.py]: I think it should be `authenticated` - the trailing `d` is missing

File: waffle/models.py
Comment (KostyaEsmukov) [inline waffle/models.py]: `_` is missing here. Looks like this is the only missing piece.

Comment (clintonb) [reply waffle/models.py]: Fixed.

**Comment by KostyaEsmukov** [comment]
The last string is translated. Thank you for this work! 

This looks complete to me. But please don't merge yet: I want to manually browse the translated interface to ensure that everything is ok. I'll let you know when I'm done (hopefully I'll be able to do this today. Tomorrow at most)

**Comment by clintonb** [comment]
Strings updated. No worries. I can't merge until @jsocol, or another user with write access, approves this pull request.

File: waffle/models.py
Comment (KostyaEsmukov) [inline waffle/models.py]: `verbose_name` is missing here

**Review by KostyaEsmukov — COMMENTED** [review]
I've checked the result and it seems to look good to me. I've made some minor modifications at Transifex.

Also `MANIFEST.in` file should include the following line for the language files to be copied with the package:

```
recursive-include waffle/locale *.mo
```

**Comment by jsocol** [comment]
> I can't merge until @jsocol, or another user with write access, approves this pull request.

I can take a look tonight

**Comment by clintonb** [comment]
@jsocol let me know if you're still able to take a look today, or when you can get to it later this week.

File: waffle/locale/en_US/LC_MESSAGES/django.po
Comment (jsocol) [inline waffle/locale/en_US/LC_MESSAGES/django.po:1]: Should these be filled out?

**Review by jsocol — APPROVED** [review]
one question but lgtm!

File: waffle/models.py
Comment (KostyaEsmukov) [inline waffle/models.py]: Shouldn't it be a plural `Groups`, like the `Users` field below?

File: waffle/locale/en_US/LC_MESSAGES/django.po
Comment (clintonb) [reply waffle/locale/en_US/LC_MESSAGES/django.po:1]: In all these years, I have never noticed that line. It doesn't show up in Transifex, and I see no method of modifying it with the `makemessages` command.

File: waffle/models.py
Comment (clintonb) [reply waffle/models.py]: Yes. Fixing...

Comment (clintonb) [reply waffle/models.py]: Pushed.

File: waffle/locale/en_US/LC_MESSAGES/django.po
Comment (KostyaEsmukov) [reply waffle/locale/en_US/LC_MESSAGES/django.po:1]: Agreed, IMO it's OK to leave it as-is. 
Here's the same line from DRF, using Transifex as well: https://github.com/encode/django-rest-framework/blob/d2994e0596c3163ac970b29dad6a61485f938045/rest_framework/locale/en/LC_MESSAGES/django.po#L1

File: waffle/models.py
Comment (KostyaEsmukov) [reply waffle/models.py]: Translated 👍

**Review by KostyaEsmukov — APPROVED** [review]
LGTM, thanks again!

**Pull request merged**
